package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "start": "backstage-cli repo start",
     "start:app-migrated": "backstage-cli repo start app-migrated backend",
+    "start:otel": "docker-compose -f ./open-telemetry/docker-compose.yaml up -d && backstage-cli repo start",
```

Comment:
if you start it in daemon mode, it outlives the start command right? as in, if you run this and then press ctrl+c to abort, does the docker-compose keep running? Makes me wonder if it's worth breaking the docker-componse out as a thing of its own that you just need to remember to run once, like `start:otel-prerequisites` or something, and then you have that going. But no worries, just an open thought